lambdas/transcode/index.py

```diff
@@ -7,14 +7,24 @@
 from t4_lambda_shared.decorator import api, validate
 from t4_lambda_shared.utils import get_default_origins, make_json_response
 
+VIDEO_FORMATS = ['mp4', 'webm']
+AUDIO_FORMATS = ['mp3', 'ogg']
+
+CONTENT_TYPES = {
+    'mp4': 'video/mp4',
+    'webm': 'video/webm',
+    'mp3': 'audio/mpeg',
+    'ogg': 'audio/ogg',
+}
```

Comment:
We could do something like this:

```suggestion
VIDEO_FORMATS =  {
    'mp4': 'video/mp4',
    'webm': 'video/webm',
}
AUDIO_FORMATS = {
    'mp3': 'audio/mpeg',
    'ogg': 'audio/ogg',
}

CONTENT_TYPES = {
    **VIDEO_FORMATS,
    **AUDIO_FORMATS,
}
```

Or we could just accept output format as content type so wouldn't need these mappings at all.

Reply:
Hm, content type is an interesting idea. It would also solve the video/audio ambiguity - e.g., could have `video/webm` and `audio/webm` if we wanted to.

Reply:
So what does everyone think about using mime types here?

Reply:
Do I understand correctly, that by using mime type you're meaning that API instead of `format=mp4` will have something like `mime=video/mp4`, and will return mp4 video according to to this mime type? Yes, it's probably better because - you're already pointing this out - we can set audio or video for the same extension.

Reply:
Yeah, exactly. (Could still call it `format`, though.)

The only argument against it I can think of is: we will be relying on someone else's standard, which might not be perfect and might not exactly fit what we want. E.g., video/mp4 might refer to multiple codecs, and we'll have no way to specify that. If we wanted to be perfect, we'd specify the container format, the audio codec, and the video codec - but that seems a bit too much.

But I think it's a pretty minor thing right now.

Reply:
> E.g., video/mp4 might refer to multiple codecs, and we'll have no way to specify that.

https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter